estate/views/estate_menus.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <menuitem id="estate_menu_root" name="Real Estates">
+        <menuitem id="estate_property_first_level_menu" name="Advertisements">
+            <menuitem id="estate_model_menu_action" name="Properties" action="estate_model_action"/>
```

Comment:
```suggestion
            <menuitem id="estate_model_menu_action" name="Properties" action="action_estate_properties"/>
```

Always name an action with `action_` followed by what it really does. Make the respective changes for all the actions